### ISSUE TITLE
[CoE Starter Kit - BUG] Internal error when setting alert for App

### ISSUE THREAD

**Issue by mdodwani** [microsoft/coe-starter-kit#9076]
### Does this bug already exist in our backlog?

- [X] I have checked and confirm this is a new bug.

### Describe the issue

I am trying to setup alert when app is shared within organization but getting following error message.
An internal error occurred while creating your trigger. If this error persists, please contact support, quoting the following information: "Missing timeaxis field"
![Set alert bug](https://github.com/user-attachments/assets/ff48b36c-c18a-4de3-abb8-f2830593d20b)

### Expected Behavior

Alert should be set

### What solution are you experiencing the issue with?

Governance

### What solution version are you using?

1.70

### What app or flow are you having the issue with?

App Deep Dive

### What method are you using to get inventory and telemetry?

Cloud flows

### Steps To Reproduce

1. Navigate to Deep dive of Power apps
2. Right click on environment and select Set alert
3. Select App Details in monitor.
4. Select Total shared with changes in condition
5. send team message and adding my teams account
6. select create button

### Anything else?

getting error message
An internal error occurred while creating your trigger. If this error persists, please contact support, quoting the following information: "Missing timeaxis field"

**Comment by Jenefer-Monroe** [comment]
Hello. We have not done any work to integrate with this feature. Can you clarify, are you in PBI? Can you please share a screenshots of these steps? Note you need to be logged in to GitHub to share screenshots.

**Comment by Jenefer-Monroe** [comment]
I see. We have not done any work to integrate with this feature so we would have to explore it before we can comment on why it isnt working. It could be an issue with the feature or it could be that the kit needs configured in a new way to get it to work. 
We are unable to do this exploration at this time due to competing priorities, but I'm happy to put it on the backlog to investigate another time.

**Comment by Jenefer-Monroe** [comment]
to backlog for consideration

**Comment by harini-2-y** [comment]
This issue is being closed as its scope is now included in a newly created consolidated enhancement request for centralized management of orphaned components (Apps, Flows, Connection References) in the CoE Starter Kit.

New Issue Reference: [[CoE Starter Kit - Feature]:https://github.com/microsoft/coe-starter-kit/issues/10337

Thank you for your contributions
